examples/src/main/java/io/kubernetes/client/examples/Example.java

```diff
@@ -26,10 +26,10 @@
 /**
  * A simple example of how to use the Java API
  * 
- * Easiest way to run this:
+ * <p>Easiest way to run this:
```

Comment:
Do these tags need to be terminated?

Reply:
In theory yes, but all browsers handle unterminated <p> correctly for legacy reasons

Reply:
In theory yes, but all browsers handle unterminated <p> correctly for legacy reasons

Reply:
The HTML specs say closing tags can be omitted for p tags. https://stackoverflow.com/a/33787857/553994 I got used to doing this in javadocs but can add them here if you’d like.